src/test/java/com/wikia/webdriver/common/enums/Mercury.java

```diff
@@ -0,0 +1,27 @@
+package com.wikia.webdriver.common.enums;
+
+/**
+ * @ownership Content X-Wing Wikia
```

Comment:
Here is a note from Patrick:
Patrick Archbold
Sep 15

to QA 
+1  but I think we should include "wikia' in the text e.g.
@ownership: Content X-Wing Wikia
My thinking behind this is that our code is visible by anyone on github not just just Wikia people.
Cheers,
Patrick

It makes sense for me, it is part of brand building in case someone starts to use our code as part of the code being open source